webui/cypress/integration/uninitialized.spec.js

```diff
@@ -1,14 +1,6 @@
 describe('Schema section', () => {
   it('Schema without bootstrap', () => {
     cy.visit(Cypress.config('baseUrl')+"/admin/cluster/schema");
-
-    cy.get('button[type="button"]:contains("Validate")').click();
-    cy.get('#root').contains('Cluster isn\'t bootstrapped yet');
```

Comment:
Why? It's not related to the topic of PR

Reply:
Changed